guides/doc-Upgrading_and_Updating/topics/upgrading_a_disconnected_satellite.adoc

```diff
@@ -3,11 +3,14 @@
 
 Use this procedure if your {ProjectServer} is not connected to the Red{nbsp}Hat Content Delivery Network.
 
+// TODO reuse common/modules/snip_using_installer_noop.adoc
 [WARNING]
 ====
 * If you customized configuration files, either manually or using a tool such as Hiera, these changes are overwritten when you enter the `{foreman-maintain}` command during upgrading or updating.
 You can use the `--noop` option with the `{foreman-installer}` command to review the changes that are applied during upgrading or updating.
```

Comment:
This feels inconsistent. First the user is told they use `{foreman-maintain}` and then that thy can call `{foreman-installer}` with `--noop`.

Additionally, later in the procedure there's an explicit "run the installer with --noop"
 (around line 240). Is this warning actually needed? Should the warning actually be removed here and moved down?

Reply:
I don't feel qualified enough to answer that. If you know a person who could answer it feel free to ping them.

Reply:
@evgeni any thoughts on this?

Reply:
On which part?
* The fact that Hiera modifications *should* survive a installer run?
* The fact that this also applies to any other invocation of the installer?
* The fact that when using `f-maintain`, the user has no chance to perform the `--noop` action with the "new" installer?

Reply:
I only thought about the last part, but now that you mention the other two ...

Reply:
So to make it actionable: given that we have explicit instructions to run with `--noop` in this guide, should we drop this specific warning?

Reply:
@evgeni should we?

Reply:
I think this warning should happen once: at the beginning of the chapter (so in upgrading_overview.adoc), any other mention is just stupidly duplicating things.

Reply:
https://github.com/theforeman/foreman-documentation/pull/1873